server/src/pages/shot/controller.js

```diff
@@ -20,6 +20,17 @@ function shouldHighlightEditIcon(model) {
   return !hasSeen;
 }
 
+function shouldShowPromo(model) {
+  if (!model.isOwner) {
+    return false;
+  }
+  const hasSeen = localStorage.hasSeenPromoDialog;
+  if (!hasSeen && model.enableAnnotations) {
+    localStorage.hasSeenPromoDialog = "1";
```

Comment:
I don't think we need to be unduly shy, we could show it several times.

Reply:
@ianb is there a recommended frequency (2-3 times?)

Reply:
I will arbitrarily say 3, because 3 is a good number.

Reply:
or interval like for next 1-2 days from first seen?

Reply:
ok cool will change it to display for 3 shot page loads